src/Sylius/Bundle/ApiBundle/Resources/config/validation/CompleteOrder.xml

```diff
@@ -16,20 +16,22 @@
                     xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/services/constraint-mapping-1.0.xsd">
 
     <class name="Sylius\Bundle\ApiBundle\Command\Checkout\CompleteOrder">
+        <constraint name="Sylius\Bundle\ApiBundle\Validator\Constraints\OrderNotEmpty">
+            <option name="groups">
+                <value>sylius_checkout_complete</value>
+            </option>
+        </constraint>
         <constraint name="Sylius\Bundle\ApiBundle\Validator\Constraints\OrderProductEligibility">
-            <option name="message">sylius.order.product_eligibility</option>
```

Comment:
Why did you delete it?

Reply:
Because the message is already defined in `OrderProductEligibility` constraint